### ISSUE TITLE
Default caching of static assets in `_nuxt` (or renamed folder)

### ISSUE THREAD

**Issue by manniL** [nuxt/framework#8956]
### Describe the feature

Since route rules are a thing now, it'd be amazing to set them up to automatically cache static assets served in `_nuxt` (or the renamed asset folder).

### Additional information

- [X] Would you be willing to help implement this feature?
- [X] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://v3.nuxtjs.org/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/framework/discussions) and [issues](https://github.com/nuxt/framework/issues).

**Comment by danielroe** [comment]
This seems like a good idea to me. Any reasons not to do this @pi0?

My only comment is that the build assets directory is configurable at runtime and so this would need some runtime configuration.

**Comment by manniL** [comment]
> My only comment is that the build assets directory is configurable at runtime and so this would need some runtime configuration.

Good catch! Will add that in the initial post ☺️

**Comment by pi0** [comment]
Exactly. There is a TODO item in nitro. Since we migrated to nitropack and made public dirs[] array and configurable, needed to add flag back (maybe as `ttl:`) and enable it back:

https://github.com/unjs/nitro/blob/8bc5ff8b4d74fffe2ac02871e32cf10b73a10bd0/src/runtime/static.ts#L85
